extension/chrome/src/js/stash_page.js

```diff
@@ -1996,7 +1996,7 @@
 		}
 	});
 
-	define('bitbucket-plugin/init', [ 'jquery' ], extensionInit);
+	require([ 'jquery' ], extensionInit);
```

Comment:
This is why the extension was not working at all: https://addons.mozilla.org/en-US/firefox/addon/bitbucket-server-reviewers/reviews/